tests/contrib/test_google.py

```diff
@@ -109,57 +109,64 @@ def test_context_local():
         assert request.headers["Authorization"] == "Bearer app2"
 
 
-def test_offline():
-    app = Flask(__name__)
-    app.secret_key = "backups"
-    goog_bp = make_google_blueprint("foo", "bar", offline=True)
-    app.register_blueprint(goog_bp)
-
-    with app.test_client() as client:
-        resp = client.get(
-            "/google",
-            base_url="https://a.b.c",
-            follow_redirects=False,
-        )
+@pytest.fixture
+def make_google_blueprint_fixture():
+    def _make_google_blueprint_fixture(**kwargs):
+        app = Flask(__name__)
+        app.secret_key = "backups"
+        goog_bp = make_google_blueprint("foo", "bar", **kwargs)
+        app.register_blueprint(goog_bp)
+
+        with app.test_client() as client:
+            return client.get(
+                "/google",
+                base_url="https://a.b.c",
+                follow_redirects=False,
+            )
```

Comment:
I don't like the idea of this being in a fixture. If we now want to do something else we can't access the `app` instance at all. I'd much rather this returns `app` instead of the response from `client.get`.

Reply:
I agree with @daenney. Since this seems like it's going to be a non-trivial amount of work to refactor these tests, could you just put them back to the repetitive state that they were in before? We can refactor them in a different pull request.

Reply:
I could have refactor this and return the app object. For the moment I reverted and put them back in their repetitive state.